journal/modernized/02-journal.adoc

```diff
@@ -9365,6 +9447,8 @@ to prevent and hinder such meetings at their peril.
 [.signed-section-signature]
 John Nicholas.
 
+--
```

Comment:
Whoa... Something is super messed up here (not your fault -- a problem in the existing document).  It looks like the whole chunk of this embedded document is missing.  See below image.  Ignore for now, when you submit your updates, I'll add the missing chunk back in.  Leaving this as a note for myself.

![missing](https://user-images.githubusercontent.com/7050938/54648448-a66be600-4a7c-11e9-9a9e-31734597f784.png)